frontend/src/TableFilter.tsx

```diff
@@ -53,12 +63,37 @@ export default function TableFilter<O extends string | MenuOption>({
                         size="sm"
                         limitTags={0}
                         placeholder={placeholder}
-                        options={options}
+                        options={specialOptions.concat(options)}
                         value={current}
                         loading={loading}
                         onFocus={() => setIsFocused(true)}
                         onBlur={() => setIsFocused(false)}
-                        onChange={(_, values) => onChange(values)}
+                        onChange={(_, values) => {
+                            const cleanedValues = values.filter((value) => {
+                                return !(
+                                    isOptionEqual(value, allOption) ||
+                                    isOptionEqual(value, emptyOption)
+                                );
+                            });
+
+                            onChange(
```

Comment:
⛏️ this might be easier to read as an extracted function with if/else instead of the chained ternaries.

Reply:
Fixed this - a little bit wordier now but a thousand times more readable 🫠